cleverpush/src/main/java/com/cleverpush/banner/AppBannerModule.java

```diff
@@ -327,85 +328,86 @@ boolean isBannerTargetingAllowed(Banner banner) {
         if (allowed && banner.getAttributes() != null && banner.getAttributes().size() > 0) {
             allowed = false;
             for (HashMap<String, String> attribute : banner.getAttributes()) {
-                String attributeId = attribute.get("id");
-                String compareAttributeValue = attribute.get("value");
-                String relationString = attribute.get("relation");
+                Double attributeId = Double.valueOf(attribute.get("id"));
```

Comment:
Revert changes

Reply:
Yes i already changes but not commited may be so i will Double to String converted